src/Components/WatchcoursePageVideoSeciton/Watchcourse.tsx

```diff
@@ -0,0 +1,209 @@
+"use client";
+import React, { useState } from "react";
+import folder from "../../../public/Icons/FolderNotchOpen.svg";
+import lecture from "../../../public/Icons/Lectures.svg";
+import duration from "../../../public/Icons/Duration.svg";
+import check from "../../../public/Icons/check.svg";
+import chevronDown from "../../../public/Icons/chevrondownminor.svg";
+import chevronUp from "../../../public/Icons/chevronup.svg";
+import  avatar1 from "../../../public/Avatars/alex.svg";
+import  avatar2 from "../../../public/Avatars/man.svg";
+import  avatar3 from "../../../public/Avatars/manAvatar.svg";
+import  avatar4 from "../../../public/Avatars/woman.svg";
+import  avatar5 from "../../../public/Avatars/womanAvatar.svg";
+
+import ReactPlayer from "react-player";
+import Link from "next/link";
+import { videos } from "@/utils/videoDatabse";
+import "./Watchcourse.scss";
+
+const Watchcourse: React.FC = () => {
+  const [currentVideo, setCurrentVideo] = useState(videos[0]);
+  const [completed, setCompleted] = useState<boolean[]>(
+    new Array(videos.length).fill(false)
+  );
+  const [expandedSections, setExpandedSections] = useState<string[]>([]);
+
+  const handleVideoChange = (videoId: number) => {
+    const video = videos.find((v) => v.id === videoId);
+    if (video) {
+      setCurrentVideo(video);
+    }
+  };
+
+  const toggleSection = (section: string) => {
+    setExpandedSections((prevState) =>
+      prevState.includes(section)
+        ? prevState.filter((sec) => sec !== section)
+        : [...prevState, section]
+    );
+  };
+
+  const handleMarkComplete = (index: number) => {
+    const updatedCompleted = [...completed];
+    updatedCompleted[index] = !updatedCompleted[index];
+    setCompleted(updatedCompleted);
+  };
+
+  const sections = Array.from(new Set(videos.map((video) => video.section)));
+
+  return (
+    <div className="watchcourse">
+      <header className="watchcourse-header">
+        <Link href="/course" className="back-button">←</Link>
```

Comment:
Burada ok'u direkt html'de eklemek yerine svg olarak figmadan ekleyelim.
- Merge Conflict'in çözülmesi gerekiyor. Master'da mevcut durumda watchCourse ve SingleCourse için yapı hazır durumda ona göre kendi componentini ekleyebilirsin.